tests/phpunit/tests/rest-api/rest-global-styles-controller.php

```diff
@@ -99,10 +189,12 @@ public function test_get_item() {
 		);
 	}
 
-	public function test_create_item() {
-		$this->markTestIncomplete();
-	}
+	public function test_create_item() {}
```

Comment:
public function test_create_item() {
	$this->markTestSkipped( 'Controller does not implement create_item().' );
}